content/docs/0.8.1/high-availability/node-failure.md

```diff
@@ -1,33 +1,40 @@
 ---
-   title: Node Failure Handling with Longhorn
-   weight: 2
+title: Node Failure Handling with Longhorn
+weight: 2
 ---
 
 ## What to expect when a Kubernetes Node fails
 
-When a Kubernetes node fails with CSI driver installed (all the following are based on Kubernetes v1.12 with default setup):
-1. After **one minute**, `kubectl get nodes` will report `NotReady` for the failure node.
-2. After about **five minutes**, the states of all the pods on the `NotReady` node will change to either `Unknown` or `NodeLost`.
-3. If you're deploying using StatefulSet or Deployment, you need to decide is if it's safe to force deletion the pod of the workload
-running on the lost node. See [here](https://kubernetes.io/docs/tasks/run-application/force-delete-stateful-set-pod/).
-    1. StatefulSet has stable identity, so Kubernetes won't force delete the Pod for the user.
-    2. Deployment doesn't have stable identity, but Longhorn is a Read-Write-Once type of storage, which means it can only attached
-    to one Pod. So the new Pod created by Kubernetes won't be able to start due to the Longhorn volume still attached to the old Pod,
-    on the lost Node.
-    3. In both cases, Kubernetes will automatically evict the pod (set deletion timestamp for the pod) on the lost node, then try to 
-    **recreate a new one with old volumes**. Because the evicted pod gets stuck in `Terminating` state and the attached Longhorn volumes 
-    cannot be released/reused, the new pod will get stuck in `ContainerCreating` state. That's why users need to decide is if it's safe to force deleting the pod.
-4. If you decide to delete the Pod manually (and forcefully), Kubernetes will take about another **six minutes** to delete the VolumeAttachment
-object associated with the Pod, thus finally detach the Longhorn volume from the lost Node and allow it to be used by the new Pod.
-    - This another six-minute is [hardcoded in Kubernetes](https://github.com/kubernetes/kubernetes/blob/5e31799701123c50025567b8534e1a62dbc0e9f6/pkg/controller/volume/attachdetach/attach_detach_controller.go#L95): 
-    If the pod on the lost node is forced deleting, the related volumes won't be unmounted correctly. Then Kubernetes will wait for this fixed timeout 
-    to directly clean up the VolumeAttachment object.
+This section is aimed to inform users of what happens during a node failure and what is expected during the recovery. The following assumes that a default setup is used with Kubernetes v1.12.
```

Comment:
Removed the sentence about Kubernetes 1.12, since now we only support Kubernetes 1.14+.

Reply:
Removed sentence